src/server/scripts/EasternKingdoms/ZulGurub/boss_jeklik.cpp

```diff
@@ -78,7 +93,17 @@ enum Events
     EVENT_SHADOW_WORD_PAIN,
     EVENT_MIND_FLAY,
     EVENT_GREATER_HEAL,
-    EVENT_SPAWN_FLYING_BATS
+    EVENT_SPAWN_FLYING_BATS,
+
+    // Bat Riders (Boss)
+    EVENT_BAT_RIDER_LOOP,
+    EVENT_BAT_RIDER_THROW_BOMB,
+
+    // Bat Riders (Trash)
+    EVENT_DEMO_SHOUT,
+    EVENT_BATTLE_COMMAND,
+    EVENT_INFECTED_BITE,
+    EVENT_UNSTABLE_CONCOTION
```

Comment:
```suggestion
    EVENT_UNSTABLE_CONCOCTION
```
Don't forget here, as well as L523, L526, L546, and L547.